assert-camunda-git-emails/check.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -eu
+
+echo "Looking at Git commits in range $GIT_RANGE"
+
+echo "All found email addresses:"
+git log --pretty=tformat:"%ae%n%ce" "$GIT_RANGE" | sort | uniq
+
+# shellcheck disable=SC2126
+NUM_VIOLATIONS=$(git log --pretty=tformat:"%ae%n%ce" "$GIT_RANGE" | \
+  grep -v "@camunda.com" | \
```

Comment:
can we make this whitelist configurable?
e.g. if I want to allow "something@renovate.com" without forking the action.

Reply:
camunda.com et al. can stay hardcoded, but the extensibility would be nice to have.

Reply:
> without forking the action
I imagined this action to be the source of truth and people to raise a PR if they want to add an exclusion. That said, I can make it configurable via an environment variable :+1:

Reply:
Please have a look at the implementation in https://github.com/camunda/infra-global-github-actions/pull/1/commits/57344c0251a4444598ffa30b0e67edd76b36666b (lacks docs yet). This requires the user to provide an expression suitable for `grep -v` directly (thus requiring `\|` separators between OR-ed parts) instead of some generic e.g. comma-separated list. The reason here is that I want to keep the complexity of this script to a minimum due to using Bash - everything requiring string manipulations I think we need to switch to something like Python.

Looking at the implementation I'm not sure if the extensibility added is worth it given the applied limitations, though. WDYT?